app/src/main/res/layout/honeycomb_button.xml

```diff
@@ -19,11 +19,13 @@
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:layout_gravity="center"
+            android:paddingStart="3dp"
+            android:paddingEnd="3dp"
             android:layout_marginStart="15dp"
             android:layout_marginEnd="15dp"
-            android:fontFamily="sans-serif-light"
+            android:fontFamily="sans-serif"
```

Comment:
minor: Q: the drop-shadows around the text – were those added by changing the `android:fontFamily`? or are they defined somewhere else?

![image](https://user-images.githubusercontent.com/203725/54202194-ba1bb900-448c-11e9-99fb-7931c4d1b70a.png)

![image](https://user-images.githubusercontent.com/203725/54202201-bee06d00-448c-11e9-803f-5926d6ac67fd.png)